examples/policies/l3/entities/from_world_deny.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumClusterwideNetworkPolicy
+metadata:
+  name: "external-lockdown"
+spec:
+  endpointSelector: {}
+  ingressDeny:
+  - fromEntities:
+    - "world"
```

Comment:
Maybe I missed but is this referenced somewhere? I wonder if it should have an `ingress: -fromEntities: - all` to ensure it doesn't just drop all traffic